pupil_src/launchables/eye.py

```diff
@@ -236,9 +236,10 @@ def consume_events_and_render_buffer():
             glfw.glfwMakeContextCurrent(main_window)
             clear_gl_screen()
 
-            glViewport(0, 0, *g_pool.camera_render_size)
-            for p in g_pool.plugins:
-                p.gl_display()
+            if 0 <= g_pool.camera_render_size[0] and g_pool.camera_render_size[1]:
```

Comment:
Why do you only check for the x value to be greater than `0`?

Reply:
In PIMonitor we used something like
```python
if 0 not in self.framebuffer_size:
```
which apparently works, but I agree that checking also for negative numbers makes sense.
Maybe
```python
if all(c > 0 for c in g_pool.camera_render_size):
```
makes sense?